types/simpl-schema/simpl-schema-tests.ts

```diff
@@ -0,0 +1,96 @@
+import SimpleSchema from 'simpl-schema';
+
+const StringSchema = new SimpleSchema({
+    basicString: {
+        type: String
+    },
+    limitedString: {
+        type: String,
+        allowedValues: ['pro', 'con']
+    },
+    regExpString: {
+        type: String,
+        regEx: SimpleSchema.RegEx.Id
+    },
+    createdAt: {
+        type: Date,
+        autoValue: () => new Date(),
+    },
+    title: {
+        type: String,
+        label: "Argument",
+        /* TODO, this does not work: "The containing arrow function captures the global value of 'this' which implicitly has type 'any'."
+        custom: () => {
+          const text = this.value;
+          if (text.length > 10) return { type: SimpleSchema.ErrorTypes.MAX_STRING, max: 10 };
+        }
+        */
+    },
+});
+
+StringSchema.validate({
+    basicString: "Test",
+    limitedString: "pro",
+    regExpString: "id"
+});
+
+    /*
```

Comment:
Did you intend to delete this comment?  Or, perhaps, turn it into further tests?

Reply:
Okay, done both!